.github/workflows/create-crd-update-pr-in-longhorn-repo.yml

```diff
@@ -46,9 +54,8 @@ jobs:
         delete-branch: true
         sign-commits: true
         signoff: true
-        author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
-        committer: Longhorn GitHub Bot <67932897+longhorn-io-github-bot@users.noreply.github.com>
```

Comment:
Do we know why the bot account with sign is not valid for DCO check?

Reply:
I have no idea for now. Although I set the committer and author to Longhorn bot, the longhorn/longhorn PR will be created by the **author** of the longhorn/longhorn-manager PR rather than the Longhorn bot when triggering the workflow by `pull_request_target`. However, using the **author** of the longhorn/longhorn-manager PR might make sense.